feedly_streams_block.py

```diff
@@ -60,10 +62,6 @@ def configure(self, context):
         self._set_user_id()
```

Comment:
@kyliedale: Almost there. But now you need to use the new id. Remove this and replace any reference to `self._user_id` with `self.user_id`